base/system/userinit/userinit.c

```diff
@@ -262,8 +215,7 @@ TryToStartShell(
         return FALSE;
     }
 
-    StartAutoApplications(CSIDL_STARTUP);
-    StartAutoApplications(CSIDL_COMMON_STARTUP);
+    // NOTE: Auto Startup codes are moved to base/shell/explorer/startup.cpp
```

Comment:
We don't even need this comment (since, according to Giannis & al, the original code was wrong to begin with).

Reply:
Deleted. OK.